runtime/extern.c

```diff
@@ -536,15 +536,35 @@ static void extern_rec(value v)
       continue;
     case Custom_tag: {
       uintnat sz_32, sz_64;
+      char * size_header;
       char * ident = Custom_ops_val(v)->identifier;
       void (*serialize)(value v, uintnat * bsize_32,
                         uintnat * bsize_64)
         = Custom_ops_val(v)->serialize;
+      const struct custom_fixed_length* fixed_length
+        = Custom_ops_val(v)->fixed_length;
       if (serialize == NULL)
         extern_invalid_argument("output_value: abstract value (Custom)");
-      write(CODE_CUSTOM);
-      writeblock(ident, strlen(ident) + 1);
-      Custom_ops_val(v)->serialize(v, &sz_32, &sz_64);
+      if (!fixed_length) {
```

Comment:
Please write `if (fixed_length == NULL)`, if only to be consistent with the previous code (2 lines up).

Reply:
Fixed.